etomo/operators/fourier/utils.py

```diff
@@ -0,0 +1,93 @@
+"""
+Functions to transform data from radon space to fourier space
+"""
+import numpy as np
+import scipy.fftpack as pfft
+
+
+def generate_locations_etomo_2D(size_x, angles):
+    """
+    This function generate the list of the samples coordinate in the k-space.
+
+    Parameters
+    ----------
+    size_x: int
+        image size along the x-axis
+    angles: np.ndarray((q))
+        array countaining the acquisition angles
+
+    Returns
+    -------
+    samples: np.ndarray((size_x * q, 2))
+        Fourier space locations generated from the given angles and data image
+        size
+   """
+    diag_x = int(np.floor(np.sqrt(2) * size_x))
+    rho = np.linspace(-0.5, 0.5, diag_x, endpoint=False)
+    for t, angle in enumerate(angles):
+        sample = np.zeros((diag_x, 2))
+        sample[:, 0] = rho * np.sin(angle * 1.0 * np.pi / 180.)
+        sample[:, 1] = rho * np.cos(angle * 1.0 * np.pi / 180.)
+        if t == 0:
+            samples = sample
+        else:
+            samples = np.concatenate([samples, sample])
+        samples = np.asarray(samples)
```

Comment:
I would recommend to use here: samples = np.concatenate( (samples, sample, axis=0) and remove the np.asarray(samples) afterwards. What do you think?

Reply:
added the axis and removed unnecessary np.asarray